src/api/uri.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @prettier
+ */
+
+/**
+ * Encode text as a URI component.
+ *
+ * FIXME: This actually calls `encodeURI` instead of `encodeURIComponent`.
+ *
+ * @param {string} str
+ * @return {string}
+ */
+function encode(str) {
+    return encodeURI(str);
```

Comment:
This should probably be changed to call `encodeURIComponent(…)` instead.

Reply:
I don't think it would be safe to make that change. It probably should have been written that way to begin with, but any change now risks changing what appears on developer.mozilla.org and I don't want to risk that since we don't have a way to test that API changes here don't affect the output on any of the thousands of MDN document, I'm very very reluctant to change anything.